DigitalLearningSolutions.Data.Migrations/202010161445_ApplyLPDefaultSPs.cs

```diff
@@ -20,12 +18,13 @@ public override void Up()
                  .WithColumn("ID").AsInt32().NotNullable().PrimaryKey().Identity()
                  .WithColumn("CandidateId").AsInt32().NotNullable().ForeignKey("Candidates", "CandidateID")
                  .WithColumn("SelfAssessmentId").AsInt32().NotNullable().ForeignKey("SelfAssessments", "ID")
-                .WithColumn("FilteredAssetID").AsInt32().NotNullable().ForeignKey("FilteredAssets", "ID")
-                .WithColumn("LaunchedDate").AsDateTime().NotNullable().WithDefault(SystemMethods.CurrentDateTime)
-                .WithColumn("LaunchCount").AsInt32().NotNullable().WithDefaultValue(1)
-                .WithColumn("CompletedDate").AsDateTime().Nullable()
-                .WithColumn("Duration").AsInt32().Nullable()
-                .WithColumn("Outcome").AsInt32().Nullable();
+                 .WithColumn("FilteredAssetID").AsInt32().NotNullable().ForeignKey("FilteredAssets", "ID")
+                 .WithColumn("LaunchedDate").AsDateTime().NotNullable().WithDefault(SystemMethods.CurrentDateTime)
+                 .WithColumn("LaunchCount").AsInt32().NotNullable().WithDefaultValue(1)
+                 .WithColumn("CompletedDate").AsDateTime().Nullable()
+                 .WithColumn("Duration").AsInt32().Nullable()
+                 .WithColumn("Outcome").AsInt32().Nullable()
+                 .WithColumn("Bookmarked").AsBoolean().Nullable();
```

Comment:
I guessed that this was missed accidentally? It was referenced in the sql script below. Let me know if this should be fixed in another way 🙂